Makefile

```diff
@@ -76,4 +76,7 @@ issues: ## Run GitHub issue analysis scripts
 	./hack/github/feature_request_reactions.py > "karpenter-feature-requests-$(shell date +"%Y-%m-%d").csv"
 	./hack/github/label_issue_count.py > "karpenter-labels-$(shell date +"%Y-%m-%d").csv"
 
-.PHONY: help dev ci release test battletest verify codegen apply delete toolchain release licenses issues
+docs: ## Serve the docs site locally
+	cd website && npm install && git submodule update --init --recursive && hugo server -l layouts/ --themesDir themes/
```

Comment:
Hmm -- `hugo server` without the flags works for me.

Reply:
ah you're right, only needed those when I was trying not to `cd website`, but turns out that adds a lot of complexity :D